pages/glossary/programmatically-hidden.md

```diff
@@ -8,15 +8,29 @@ input_aspects:
   - DOM tree
 ---
 
-An HTML element is _programmatically hidden_ if either it has a [computed][] CSS property `visibility` whose value is not `visible`; or at least one of the following is true for any of its [inclusive ancestors][] in the [flat tree][]:
+An HTML element is _programmatically hidden_ if any of the following is true:
```

Comment:
Not relevant to this PR, but I thought I'd share. I recently discovered that you can't hide area elements with display:none. Probably not something we need to worry about, I just thought it was interesting.

Reply:
`area`  elements are not really rendered (and do have a default `display: none`: [HTML rendering guidelines](https://html.spec.whatwg.org/multipage/rendering.html#hidden-elements)).
It is the associated image which is rendered, but neither the `map` nor the `area` is (they are mostly abstractions for interactions).